spec/c08chalenges/non_overlapping_intervalSpec.js

```diff
@@ -0,0 +1,23 @@
+const insertInterval = require('../../src/c08challenges/non_overlapping_interval')
+
+describe('No overlapping intervals', function () {
+    it('Should find the correct position of non-overlaping intervals', function () {
+
+        let array = [[1, 3], [6, 9]]
```

Comment:
Give this variable a name that fits the context of the problem. Like `intervals`.